interface-definitions/dhcp-relay.xml.in

```diff
@@ -9,7 +9,54 @@
           <priority>910</priority>
         </properties>
         <children>
-          #include <include/generic-interface-multi-broadcast.xml.i>
+          <leafNode name="interface">
```

Comment:
Please keep `#include <include/generic-interface-multi-broadcast.xml.i>` and add a `vyos.base.Warning()` message inside the Python script when the old command is used.